wolfcrypt/src/random.c

```diff
@@ -3476,7 +3476,15 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
     }
 #elif defined(WOLFSSL_RENESAS_FSPSM) || \
           defined(WOLFSSL_RENESAS_FSPSM_CRYPTONLY)
+
+   #if defined(WOLFSSL_RENESAS_SCEPROTECT)
     #include "r_sce.h"
+    #define R_RANDOM_GEN(b) R_SCE_RandomNumberGenerate(b)
```

Comment:
Please adjust spacing in this section to match coding standards - tabs should be 4 spaces.  Looks like these several lines use 1-space in after #if/elif/etc.

Reply:
Thanks, fixed.